### PR TITLE
🌎 Add alternative urls to page frontmatter and JATS self-uri

### PR DESCRIPTION
**Pull request by fwkoch**
This also has a small tweak where departments are removed from JATS funding info.

### CODE REVIEW

File: docs/frontmatter.md [hunk 1]
```diff
@@ -88,6 +88,9 @@ The following table lists the available frontmatter fields, a brief description
 * - `subtitle`
   - a string (max 500 chars)
   - page only
+* - `alternatives`
+  - alternative URLs and descriptions for the page content (see [](#alternatives))
+  - page only
```
Comment (rowanc1) [inline docs/frontmatter.md:93]: Should this be page only? If we have a zenodo repository that is an alternate, that is for the project.

**Review by rowanc1 — APPROVED** [review]
Looks good, wonder if we open up to project as well?

File: docs/frontmatter.md (hunk 1 above)
Comment (fwkoch) [reply docs/frontmatter.md:93]: Cool, yeah this is an easy change. Should the project values read through to the page though? i.e. should the zenodo repository be an alternative for every single page that doesn't have a different alternative defined?

Comment (rowanc1) [reply docs/frontmatter.md:93]: Maybe not? Certainly not in JATS. They seem like different things, one for the project and one for the page.

Comment (rowanc1) [reply docs/frontmatter.md]: @fwkoch should we actually go the other way and only allow this on projects to start with? That seems a bit easier to handle. Right now the links from projects get lost when converting to JATS.

**Review by rowanc1 — CHANGES_REQUESTED** [review]
Let's let this one sit for a bit, there might be other ways to do it in talking through with @fwkoch.

**Pull request closed without merging**